src/routes/City.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { API_EDPOIONTS } from '../helper/variables';
+import useFetch from '../hooks/useFetch';
+import styles from '../styles.scss';
+
+function City() {
+  const [cityInfo, setCityInfo] = useState([]);
+  const [cityName, setCityName] = useState('');
+
+  const cityId = useParams().id;
+  useEffect(() => {
+    (async () => {
+      const responseName = await fetch(
+        `http://localhost:3333/pfa.foreca.com/api/v1/location/${cityId}`
+      );
+      const reqDataName = await responseName.json();
+      setCityName(reqDataName.name);
+      const responseInfo = await fetch(
+        `http://localhost:3333/pfa.foreca.com/api/v1/current/${cityId}`
+      );
+      const reqDataInfo = await responseInfo.json();
+      setCityInfo(reqDataInfo.current);
+    })();
+  }, [cityId]);
```

Comment:
I'm wondering if you'll be able to use your query hook here

Reply:
![image](https://user-images.githubusercontent.com/101064758/195294393-6de64f42-17a4-4e5c-a7d6-4596143e6eb5.png)
that's how look  my hook logic